src/popup/popup.js

```diff
@@ -9,6 +9,10 @@ var Popup = Popup || {};
       code: "window.highlightComponent = window.highlightComponent || new HighlightComponent; undefined;"
     });
 
+    chrome.tabs.executeScript(null, {
+      code: "window.designModeComponent = window.designModeComponent || new DesignModeComponent; undefined;"
```

Comment:
I know this is copied from the `highlightComponent` block, but why do we have `undefined;` at the end? Just curious!

Reply:
I really don't know.. there must be a reason.

Reply:
I think the answer is in the comment of https://github.com/alphagov/govuk-browser-extension/commit/f445fe662925b2d048c3b2cee597d3b09ee5c8a4

Reply:
Ah! "* Add `undefined;` at the end of all content scripts, since they need to return something"